ROADMAP.md

```diff
@@ -3,69 +3,92 @@
+  - Extending project descriptor to support process types, labels, and runtime env vars.
+  - Support for Inline Buildpacks.
 - **Personas**:
   - Application Developers
   - Buildpack Authors
-- **Status**: Not started
 
-### Spec Refactor
+### Integration with the Cloud Native Ecosystem
+
+![status-started](https://img.shields.io/badge/status-started-green)
+[![issues](https://img.shields.io/github/issues-search?logo=github&label=roadmap%2Fcloud-ecosystem&query=org%3Abuildpacks+label%3Aroadmap%2Fcloud-ecosystem)](https://github.com/search?q=org%3Abuildpacks+label%3Aroadmap%2Fcloud-ecosystem)
+
 - **Outcomes**: 
-  - A restructured specification for Cloud Native Buildpacks that is more approachable, and easier to ammend.
+  - A better out-of-the box Kubernetes and Docker integration story
 - **Personas**:
+  - Application Developers
   - Platform Authors
+
+### Onboarding
+
+![status-started](https://img.shields.io/badge/status-started-green)
+[![issues](https://img.shields.io/github/issues-search?logo=github&label=roadmap%2Fonboarding&query=org%3Abuildpacks+label%3Aroadmap%2Fonboarding)](https://github.com/search?q=org%3Abuildpacks+label%3Aroadmap%2Fonboarding)
+
+- **Outcomes**:
+  - Revisit our documentation so it's easier to find the content needed, includes a reference section, and backfill missing topics.
+  - Improve the onboarding experience by provide more getting started materials.
+  - Provide better discoverability of components via the Buildpacks Registry.
+- **Personas**:
+  - Application Developers
   - Buildpack Authors
   - Contributors
-- **Status**: Not started
+  - Operators
+  - Platform Authors
+
+### Operational Efficiency
+
+![status-started](https://img.shields.io/badge/status-started-green)
+[![issues](https://img.shields.io/github/issues-search?logo=github&label=roadmap%2Foperations&query=org%3Abuildpacks+label%3Aroadmap%2Foperations)](https://github.com/search?q=org%3Abuildpacks+label%3Aroadmap%2Foperations)
+
+- **Outcomes**:
+  - TBD
+- **Personas**:
+  - Contributors
+
+### Spec Simplification
+
+![status-started](https://img.shields.io/badge/status-started-green)
+[![issues](https://img.shields.io/github/issues-search?logo=github&label=roadmap%2Fspec&query=org%3Abuildpacks+label%3Aroadmap%2Fspec)](https://github.com/search?q=org%3Abuildpacks+label%3Aroadmap%2Fspec)
 
-### Configurability
 - **Outcomes**: 
-  - Better escape hatches, flexibility, and ways of configuring buildpack builds. 
-  - Extending project descriptor to support process types, labels, and runtime env vars.
-  - Support for Inline Buildpacks.
+  - A restructured specification for Cloud Native Buildpacks that is more approachable, and easier to ammend.
 - **Personas**:
-  - Application Developers
   - Buildpack Authors
-- **Status**: Not started
-- **RFCs**:
-  - [RFC-0048](https://github.com/buildpacks/rfcs/blob/main/text/0048-inline-buildpack.md)
+  - Contributors
+  - Platform Authors
 
 ### Stack Buildpacks
```

Comment:
@sclevine what's a better name for this now?

Reply:
@sclevine Any update on this?

Reply:
I wonder if this is better served under "Configurability" since that's ultimately what we're trying to achieve.